src/flag_gems/fused/fused_add_rms_norm.py

```diff
@@ -14,37 +14,44 @@
 @libentry()
 @triton.jit(do_not_specialize=["eps"])
 def fused_add_rms_norm_kernel(
-    X,  # pointer to the input
-    R,  # pointer to the residual
-    W,  # pointer to the weights
-    x_stride_r,  # how much to increase the pointer when moving by 1 row
-    x_stride_c,  # how much to increase the pointer when moving by 1 col
+    in_ptr,  # pointer to the input
+    re_ptr,  # pointer to the residual
```

Comment:
I think ppl would appreciate more verbatim naming style. Specifically, `inp_ptr`, `residual_ptr` look much more clear to me.